src/grpc/infer_handler.cc

```diff
@@ -168,7 +168,9 @@ TRITONSERVER_Error* InferGRPCToInput(
     const std::shared_ptr<SharedMemoryManager>& shm_manager,
     const inference::ModelInferRequest& request,
     std::list<std::string>* serialized_data,
-    TRITONSERVER_InferenceRequest* inference_request);
+    TRITONSERVER_InferenceRequest* inference_request,
+    std::vector<std::shared_ptr<const SharedMemoryManager::SharedMemoryInfo>>*
+        ref_shm_regions);
```

Comment:
Can you double check if line 161-173 is needed? The function has been declared in the `infer_handler.h` and I am not sure why it's done again here.

Reply:
Good catch. It seems that these methods are mistakenly declared again here. I have removed them.